src/Microsoft.Framework.TestHost/ProjectCommand.cs

```diff
@@ -3,46 +3,89 @@
 
 using System;
 using System.Linq;
+using System.IO;
+using System.Reflection;
 using System.Threading.Tasks;
 using Microsoft.Framework.Runtime;
 using Microsoft.Framework.Runtime.Common.DependencyInjection;
+using Microsoft.Framework.Runtime.CommandParsing;
+using Microsoft.Framework.Runtime.Common;
+using Microsoft.Framework.Runtime.Infrastructure;
 
 namespace Microsoft.Framework.TestHost
 {
-    public class ProjectCommand
+    public static class ProjectCommand
     {
         public static async Task<int> Execute(
             IServiceProvider services, 
             Project project,
```

Comment:
We need to stop using this as well and parse the JSON by hand. Possibly using the new JSON parser in the runtime.